src/Umbraco.Web.UI.Client/src/views/common/dialogs/help.html

```diff
@@ -1,13 +1,13 @@
 <div class="umb-panel" ng-controller="Umbraco.Dialogs.HelpController">
     <div class="umb-panel-header">
-        <h1 class="headline">Help</h1>
+        <h1 class="headline"><localize key="sections_help">Help</localize></h1>
         <small class="umb-version">Umbraco version {{version}}</small>
     </div>
 	
 	<div class="umb-panel-body umb-scrollable">
 		<div class="tab-content umb-control-group">
 			<div class="umb-pane">
-				<h5>Help topics for: {{section}}</h5>
+				<h5><localize key="help_helpTopicsFor">Help topics for</localize>: {{section}}</h5>
```

Comment:
Yes, those can be localized... but would probably need to be done in the JS since i don't think the localize directive supports the interpolation of sections_{{section}} as an attribute